rootfs/Dockerfile

```diff
@@ -1,20 +1,20 @@
-FROM quay.io/deis/base:v0.3.6
+FROM minio/mc:RELEASE.2018-11-06T01-12-20Z as mc
 
-RUN adduser --system \
-	--shell /bin/bash \
-	--disabled-password \
-	--home /home/minio \
-	--group \
-	minio
+FROM minio/minio:RELEASE.2018-11-06T01-01-02Z
+
+COPY --from=mc /usr/bin/mc /usr/bin/mc
+
+RUN adduser \
```

Comment:
Should probably be noted that this switches the resulting image from ubuntu to alpine.

(I like this move very much as it probably trims the size tremendously, but it should be noted somewhere as other tools familiar to maintainers/devs/users might break or be unavailable.)

Reply:
We debated this on slack ([slack.teamhephy.com](https://slack.teamhephy.com/)) and we are going with official docker images where we can.